catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/SolrFilterDelegate.java

```diff
@@ -65,6 +65,214 @@
 
     private static final GeometryFactory GEOMETRY_FACTORY = new GeometryFactory();
 
+    @Override
+    public SolrQuery include() {
+        return super.include();
```

Comment:
This is effectively the same as not overriding these methods.  I would not add any method that you are not explicitly overriding. !fix